rero_ils/modules/items/api/record.py

```diff
@@ -428,7 +428,7 @@ def get_note(self, note_type):
         """Return an item note by its type.
 
         :param note_type: the type of note (see ``ItemNoteTypes``)
-        :return the content of the note, None if note type is not found
+        :return the content of the note, "None" if note type is not found
```

Comment:
Does this work?

Reply:
I need to ask a developer on this. @jma or @rerowep, what do you think?

Reply:
@comradekingu welcome to this project and many thanks for your contribution. Yes this works, but it can be rewrite as:
```python
# here we have only one note of a specific type
return notes[0] if notes else None
```
or better
```python
return next((note.get('content') for note in self.notes
                     if note.get('type') == note_type), None)
```

or

```python
return next((note.get('content') for note in filter(lambda n: n['type'] == note_type, self.notes)), None)
```